.env.example

```diff
@@ -4,6 +4,7 @@ APP_KEY=
 APP_TIMEZONE=UTC
 APP_URL=http://panel.test
 APP_LOCALE=en
+APP_INSTALLED=false
```

Comment:
I really don't like this being necessary for a default app.

Anybody who copies this .env over will automatically have the installer enabled which might be a problem.

Reply:
We can revisit this. For now i'm going to merge it so the installer isn't broken for every new install.